templates/registration/logged_out.html

```diff
@@ -0,0 +1,6 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Logged out</h1>
+
+  <a href="{% url 'login' %}">Click here and login again!</a>
+{% endblock %}
```

Comment:
Oh no, this `{% endblock %}` is [not a newline-terminated string](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)!
Same in `login.html`.

Reply:
OH NO!!!!
thank you :)))))))

Reply:
OH NO!!!!
thank you :)))))))